task/__init__.py

```diff
@@ -54,6 +54,7 @@
 # Servers which have public images
 PUBLIC_STORES = [
     "https://images-cockpit.apps.ci.centos.org/",
+    "https://images-frontdoor.apps.ocp.ci.centos.org/",
```

Comment:
Please move this up, as it should be the primary store now. Otherwise image-upload will prefer uploading to ocp3.